plugins/account_history/plugin.cpp

```diff
@@ -135,10 +133,10 @@ if (options.count(name)) { \
     };
 
     DEFINE_API(plugin, get_account_history) {
-        CHECK_ARG_SIZE(3)
-        auto account = args.args->at(0).as<std::string>();
-        auto from = args.args->at(1).as<uint64_t>();
-        auto limit = args.args->at(2).as<uint32_t>();
+        GOLOS_CHECK_ARGS_COUNT(args.args, 3);
+        GOLOS_DECLARE_PARAM(account, args.args->at(0).as<std::string>());
+        GOLOS_DECLARE_PARAM(from, args.args->at(1).as<uint64_t>());
+        GOLOS_DECLARE_PARAM(limit, args.args->at(2).as<uint32_t>());
```

Comment:
btw, here: [plugins/debug_node/api_helper.hpp](https://github.com/GolosChain/golos/blob/master/plugins/debug_node/include/golos/plugins/debug_node/api_helper.hpp) we have macros to simplify api parameters initialisation. It's quite complex and used only in `debug_node` now.